CMakeLists.txt

```diff
@@ -150,11 +148,6 @@ endif()
 
 include(NetdataCompilerFlags)
 
-check_c_compiler_flag("-fexceptions" HAVE_FEXCEPTIONS)
-if (NOT HAVE_FEXCEPTIONS)
-    message(FATAL_ERROR "Missing required compiler flag: -fexceptions.")
-endif()
-
```

Comment:
For anyone reviewing this PR, this is being removed here because the same check is already being done by the `add_required_compiler_flag("-fexceptions")` call in the NetdataCompiler Flags module.